tests/test_security_path_traversal.py

```
@@ -204,3 +204,19 @@ def test_run_tailer_path_validation(tmp_path):
 
     with pytest.raises(ValueError, match="run_id"):
         tailer.tail_run("..")
+
+def test_statsdb_rebuild_path_validation(tmp_path):
```

Ruff (0.14.14)

[warning] Unused function argument: `tmp_path`

(ARG001)